### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chStaiger**
If sites-enabled contains a `default` page, one needs to remove that. Otherwise nginx will not find the REST API endpoints.

### CODE REVIEW

**Review by korydraughn — APPROVED** [review]
Looks good to me.

Let's wait for one more approval before updating the commit message.

**Review by alanking — APPROVED** [review]
Looks good to me. Let's change the commit title to "[#25] Update README.md" such that the issue is linked. Thanks!

**Comment by alanking** [comment]
Forgot that I have powers now. :) Thanks again

**Pull request merged**
